roles/ui/files/FWO.UI/Pages/Reporting/Report.razor

```diff
@@ -29,22 +30,24 @@
         {
             <div class="p-3">
                 <h6 class="text-left">@(userConfig.GetText("recert_parameter"))</h6>
+                    <h7 class="text-left">@(userConfig.GetText("due_within")):</h7>
+                        <input  type="text" class="form-control form-control-sm" @bind="userConfig.RecertificationDisplayPeriod" />
                     <h7 class="text-left">@(userConfig.GetText("owner"))</h7>
-                    <Dropdown ElementType="FwoOwner" ElementToString="@(o => o.Name)"
-                        Value="selectedOwner" ValueChanged="OwnerChanged" Elements="ownerList">
-                        <ElementTemplate Context="owner">
-                            @owner.Name
-                        </ElementTemplate>
-                    </Dropdown>
-                    @if (selectedOwner.Name != "")
-                    {
-                        recertWithoutOwner = false;
-                    }
-                <br>
+                        <Dropdown ElementType="FwoOwner" ElementToString="@(o => o.Name)"
+                            Value="selectedOwner" ValueChanged="OwnerChanged" Elements="ownerList">
+                            <ElementTemplate Context="owner">
+                                @owner.Name
+                            </ElementTemplate>
+                        </Dropdown>
+                        @if (selectedOwner.Name != "")
+                        {
+                            recertWithoutOwner = false;
+                        }
+                        <br>
                         <div class="form-check">
-                            <input class="form-check-input" type="checkbox" id="recertOverdue" disabled
+                            <input class="form-check-input" type="checkbox" id="recertOverdue"
                                 @bind="recertOverdueOnly"
-                                @oninput="@(async () => { recertOverdueOnly = !recertOverdueOnly; })" />
+                                @oninput="@(async () => { recertOverdueOnly = !recertOverdueOnly; userConfig.RecertificationDisplayPeriod = 0; SyncFilterToDisplay(); })" />
```

Comment:
So the original config value is forgotten for the rest of the session?

Reply:
thanks for spotting this testing option - removed it